File: Zeze/Raft/RocksRaft/LogSet.cs
using System;
using System.Collections.Generic;
using System.Collections.Immutable;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using Zeze.Serialize;

namespace Zeze.Raft.RocksRaft
{
	public class LogSet<V> : LogBean
	{
		internal ImmutableHashSet<V> Value { get; set; }

		public override void Collect(Changes changes, Bean recent, Log vlog)
		{
			throw new Exception($"Collect Not Implement.");
		}

	}
}

File: ZezeJava/ZezeJava/src/main/java/Zeze/Raft/RocksRaft/LogSet.java
package Zeze.Raft.RocksRaft;

public abstract class LogSet<V> extends LogBean {
	private org.pcollections.PSet<V> value;

	public LogSet(int typeId) {
		super(typeId);
	}

	public final org.pcollections.PSet<V> getValue() {
		return value;
	}

	public final void setValue(org.pcollections.PSet<V> value) {
		this.value = value;
	}

	@Override
	public void collect(Changes changes, Bean recent, Log vlog) {
		throw new UnsupportedOperationException("Collect Not Implement.");
	}
}
